MakeHist/mumu.C

```diff
@@ -162,15 +154,36 @@ void mumu::Loop(TString sample_name)
                                        //Number of b-jet
                                        Nbjet ++;
                                    }
+                                   else{
```

Comment:
You can actually do the following:
`else if(jet_pt[NJ] > 30){`
That will spare you one curly bracket. Just a suggestion.

Reply:
I will update the codes according to your opinion.